generators/component/templates/functions.php

```diff
@@ -2,17 +2,18 @@
 
 namespace Flynt\Components\<%= nameUpperCamelCase %>;
 
-use Flynt\Api;
+use Flynt\FieldVariables;
 use Flynt\Utils\Options;
 
 add_filter('Flynt/addComponentData?name=<%= nameUpperCamelCase %>', function ($data) {
     return $data;
 });
 
-Api::registerFields('<%= nameUpperCamelCase %>', [
-    'layout' => [
-        'name' => '<%= nameLowerCamelCase %>',
-        'label' => '<%= namePrettySplit %>',
+function getACFLayout()
+{
+    return [
+        'name' => '<%= nameUpperCamelCase %>',
+        'label' => '<%= namePretty %>',
```

Comment:
why were these two changed? the name should be lowerCamelCase, no?
also the label is now wrapped in a `__()`. could you include that, please?

Reply:
Correct! I fixed the issues